Digipost.Signature.Api.Client.Direct.Tests/DirectClientTests.cs

```diff
@@ -131,5 +132,25 @@ public async Task Throws_too_eager_polling_exception_on_too_many_requests()
                 await Assert.ThrowsAsync<TooEagerPollingException>(async () => await directClient.GetStatusChange().ConfigureAwait(false)).ConfigureAwait(false);
             }
         }
+
+        public class RequestNewRedirectUrl : DirectClientTests
+        {
+            [Fact]
+            public async Task Can_request_new_redirect_url()
+            {
+                var directClient = new DirectClient(GetClientConfiguration())
+                {
+                    HttpClient = GetHttpClientWithHandler(new FakeHttpClientHandlerForNewRedirectUrlResponse())
+                };
+
+                await directClient.RequestNewRedirectUrl(
+                    new SignerResponse(
+                        new PersonalIdentificationNumber("01013300001"),
+                        new Uri("http://someRedirectUrl.no"),
+                        new Uri("http://someSignerUrl.no")
```

Comment:
Todo: Lag klasse som inkapsulerer dette feltet, for å unngå at vi må sende inn alt.

Reply:
`WithSignerUrl` som i Java-klienten